command.go

```diff
@@ -20,6 +20,12 @@ type ArgTransformer struct {
 
 	// Function is used to transform the argument. The function should error if this is not possible.
 	Function func(ctx *Context, Arg string) (interface{}, error)
+
+	// Default is the value the arg will have if it isn't supplied by the user.
+	// This will not be used in case of an error from the transformer function, only when no argument is passed by the user.
+	// Either a slice or a single value can be passed here and it will be handled accordingly.
```

Comment:
If a slice is passed as default, in my opinion there should be no special behaviour. This is because it should be treated as the argument at that specific index or its behaviour comes unpredictable (especially when things like the greedy parser don't use this different behaviour). Mostly LGTM though.